### PR TITLE
enh: update native tmq api

### PR DESCRIPTION
**Pull request by sunpe**

### CODE REVIEW

File: taos/cinterface.py [hunk 1]
```diff
@@ -1036,10 +1036,10 @@ def taos_stmt_affected_rows(stmt):
 
 
 def taos_schemaless_insert(
-    connection: c_void_p,
-    lines: Union[List[str], Tuple[str]],
-    protocol: SmlProtocol,
-    precision: SmlPrecision,
+        connection: c_void_p,
```
Comment (sangshuduo) [inline taos/cinterface.py:1039]: unnecessary change

Comment (sunpe) [reply taos/cinterface.py:1039]: it's re-formatted by pycharm.

Comment (sangshuduo) [reply taos/cinterface.py:1039]: how about setting its indent to 4 spaces?

Comment (sunpe) [reply taos/cinterface.py:1039]: The fields are parameters of the function and are wrapped, I think they should be indented twice, so I think this re-formatting is ok.

**Review by sangshuduo — APPROVED** [review]
